src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -101,20 +102,6 @@ MavlinkMissionManager::~MavlinkMissionManager()
 	orb_unadvertise(_offboard_mission_pub);
 }
 
-unsigned
-MavlinkMissionManager::get_size()
```

Comment:
Since the object is now entirely in the receiver thread, the main thread does not know about it anymore, and thus does not include it in the rate calculations.
I was a bit concerned about this too, but my tests showed that it still works well (with fast and slow links).
With regard to https://github.com/PX4/Firmware/issues/7250, we could generally reduce the streams, when a mission/param transfer or log download is in progress.